src/hooks/useSnack.jsx

```diff
@@ -0,0 +1,13 @@
+import { useContext } from "react";
+
+import SnackbarContext from "../context/SnackbarContext";
+
+const useSnack = () => {
+  const { showSnack } = useContext(SnackbarContext);
+
+  return {
+    showSnack,
+  };
+};
+
+export default useSnack;
```

Comment:
can we rename this to a js file, so it doesn't sort to the components?

Reply:
yes thats right